src/common/init.ts

```diff
@@ -9,5 +9,10 @@ export function initializeI18Next() {
 }
 
 export const init = () => {
+  try {
+    require('../../../../../plugins');
```

Comment:
There is probably a better way to do this so please let me know if you can think of one. We want to load plugins if they exist at `<projectRoot>/plugins`. When this file in the module is installed it is _likely_ located at `<projectRoot>/node_modules/@lifeomic/react-native-sdk/src/common/init.js`